test/connectors/hip/httpParsers/CalculationListLegacyHttpParserSpec.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.hip.httpParsers
+
+import constants.CalculationListTestConstants._
+import connectors.hip.httpParsers.CalculationListLegacyHttpParser.CalculationListReads
+import models.calculationList.CalculationListResponseModel
+import models.hipErrors.{BadGatewayResponse, CustomResponse, ErrorResponse, UnexpectedJsonResponse, UnexpectedResponse}
+import play.api.http.Status
+import play.api.http.Status.{BAD_REQUEST, INTERNAL_SERVER_ERROR, NOT_FOUND, SERVICE_UNAVAILABLE, UNAUTHORIZED}
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.HttpResponse
+import utils.TestSupport
+
+class CalculationListLegacyHttpParserSpec extends TestSupport {
+  "The CalculationListLegacyHttpParser" should {
+    "return an instance of CalculationList" when {
+      "HTTP response is 200 and matches schema" in {
+        val httpResponse: HttpResponse = HttpResponse(Status.OK, jsonResponseFull, Map.empty)
+
+        val expected: Either[Nothing, CalculationListResponseModel] = Right(calculationListFull)
+        val result: CalculationListLegacyHttpParser.HttpGetResult[CalculationListResponseModel] = CalculationListReads.read("", "", httpResponse)
+
+        result shouldEqual expected
```

Comment:
sorry if pedantic but can these be shouldBe instead of shouldEqual just for consistency.